prov/cxi/configure.m4

```diff
@@ -46,11 +46,18 @@ AC_DEFUN([FI_CXI_CONFIGURE],[
 		[
 			AC_CHECK_HEADER(cxi_prov_hw.h,
 				[],
-				[cxi_happy=0])
+				[cxi_happy=0],
+                [AC_INCLUDES_DEFAULT
```

Comment:
We shouldn't take these m4 changes. 

The problem that these changes are working around are only present until the next release (2.2.0), and so I'd like to prevent a regression by introducing these and having someone remove the headers again by accident. It is a useful check that these failed since it indicated a lower level problem. 

That said, if this is really critical, then we could discuss it further.

Reply:
I can remove those :-) added them accidentally